### PR TITLE
Release 0.13.4

### PR DESCRIPTION
**Pull request by pksunkara**
Please go through the PR commit by commit. I have included more than 1 tasks here.

- [ ] Merge and publish https://github.com/apiaryio/swagger-zoo/pull/44
- [ ] Rebuild CI

### CODE REVIEW

File: src/parser.js [hunk 1]
```diff
@@ -1354,6 +1361,13 @@ export default class Parser {
     _.forEach(params, (parameter, index) => {
       this.withPath('parameters', index, () => {
         let member;
+        const format = parameter.collectionFormat || 'csv';
+
+        // Adding a warning if format is not supported
+        if (['multi', 'csv'].indexOf(format) === -1) {
```
Comment (kylef) [inline src/parser.js]:
```diff
- ['multi', 'csv'].indexOf(format) === -1
+ ['multi', 'csv'].includes(format)
```

File: src/parser.js [hunk 2]
```diff
@@ -1354,6 +1361,13 @@ export default class Parser {
...
+          this.createAnnotation(annotations.DATA_LOST, this.path,
+            `Parameters of collection format '${format}' are not yet supported`);
```
Comment (kylef) [inline src/parser.js]: I'd write this message as less of a promise "not yet supported". This message would imply we will support it, which we may never do.

```diff
- `Parameters of collection format '${format}' are not yet supported`);
+ `Parameters of collection format '${format}' are not supported`);
```

**Comment by honzajavorek** [comment]
I see many changes and no tests 👎

**Comment by pksunkara** [comment]
@honzajavorek The tests are in swagger-zoo PR linked in the description.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+# 0.13.4
+
+## Enhancements
+
+- Added support for `externalDocs`.
+- Parser will not give warning about unsupported collection formats in parameters.
```
Comment (honzajavorek) [inline CHANGELOG.md]: Why parser won't warn about them if they're unsupported?

Comment (kylef) [reply CHANGELOG.md]: Oh I think this is a mistake in changelog the following warning is emitted:

> Parameters of collection format '${format}' are not supported

Comment (pksunkara) [reply CHANGELOG.md]: Yup, it was mistype of `now` to `not`.

**Pull request merged**